.gitignore

```diff
@@ -1,6 +1,7 @@
-/_build/
```

Comment:
I have a preference for more specific patterns (these were anchored to the project root), but whatever.